app/frontend/components/pages/dashboard/dashboardPage.tsx

```diff
@@ -139,12 +122,12 @@ const AdvancedPage = ({isViewportSmall}: {isViewportSmall: boolean}) => {
   )
 }
 
-const AccountsPage = ({isViewportSmall}) => {
+const AccountsPage = ({screenSize}: {screenSize: ScreenSize}) => {
   const subTabs = [SubTabs.ACCOUNTS]
   const defaultSubTab = SubTabs.ACCOUNTS
   return (
     <Fragment>
-      {isViewportSmall ? (
+      {screenSize < ScreenSize.DESKTOP ? (
```

Comment:
comparing enums with `<` is kind of fragile. I would perhaps preserve the isViewportSmall function but it would take the screenSize as an argument and I would make it exportable from useViewport.js, perhaps renaming that file to just "viewPort"

Reply:
The motivation for this kind of comparison was that different pages have a different "splitting point" for their logic. For example, dashboardPage needs to have one layout for (mobile, tablet) together and another layout for the desktops. On the other hand, loginPage needs to have one layout for mobile and the other for (tablet, desktop) together.

I would therefore shoot for two functions to retain these "intervals". The first would be the one you mentioned but renamed
```
  function isSmallerThanDesktop(screenSize: ScreenSize) {
    return screenSize in [ScreenSize.MOBILE, ScreenSize.TABLET]
}
```
and the other one would work for the other splitting point (I believe we won't need more splits)
```
  function isBiggerThanMobile(screenSize: ScreenSize) {
    return screenSize in [ScreenSize.TABLET, ScreenSize.DESKTOP]
}
```

What do you think?

Reply:
I see, makes sense, what you suggest looks good to me